product_docs/docs/biganimal/release/overview/03a_region_support.mdx

```diff
@@ -60,6 +62,7 @@ When using AWS, you can create clusters in the following regions:
   - Europe 1 (Ireland)
   - Europe 2 (London)
 - Asia Pacific (AP)
```

Comment:
What ordering model are you using in this section? Looks like AWS orders by city. If we used that model Tokyo is out of order. If we are ordering alphabetically by the naming conventions we are using, then South 1 (Mumbai) seems out of place. I don't think we need to adhere to AWS's model but I think we need to be consistent with whatever ordering model we are using. Maybe we are already and I just don't get the model.

Reply:
Rearranged as below:

## Supported Azure regions 

When using Azure, you can create clusters in the following regions:

**Single node and high availability clusters**

- Asia Pacific, Japan (APJ)
  - Australia East (New South Wales)
  - Central India (Pune)
  - Japan East (Tokyo, Saitama)
  - South East Asia (Singapore)
- Europe, Middle East, Africa (EMEA)
  - France Central (Paris)
  - North Europe (Ireland)
  - Norway East (Oslo)
  - UK South (London)
  - West Europe (Netherlands)
- North America (NA):
  - Canada Central (Toronto)
  - Central US (Iowa)
  - East US (Virginia)
  - East US 2 (Virigina)
  - South Central US (Texas)
  - West US 2 (Washington)
  - West US 3 (Arizona)
- South America (SA):
  - Brazil South (Sao Paulo)

**Extreme high availability clusters (beta)** 

- Asia Pacific, Japan (APJ)
  - Australia East (New South Wales)
  - Central India (Pune)
  - Japan East (Tokyo, Saitama)
  - South East Asia (Singapore)
- Europe, Middle East, Africa (EMEA)
  - North Europe (Ireland)
  - UK South (London)
- North America (NA):
  - Canada Central (Toronto)
  - Central US (Iowa)
  - East US (Virginia)
  - East US 2 (Virigina)
  - West US 2 (Washington)
  - West US 3 (Arizona)

## Supported AWS regions 

When using AWS, you can create clusters in the following regions:

**Single node, high availability, and extreme high availability (beta) clusters**

- Asia Pacific (AP)
  - Asia Pacific Northeast 1 (Tokyo)
  - Asia Pacific Northeast 2 (Seoul)
  - Asia Pacific South 1 (Mumbai)
  - Asia Pacific Southeast 1 (Singapore)
- Europe (EU)
  - Europe 1 (Ireland)
  - Europe 2 (London)
- United States (US):
  - US East 1 (N. Virginia)
  - US East 2 (Ohio)
  - US West 2 (Oregon)

Reply:
What ordering is used in the UI and in the script outputs?

Reply:
I found this snippet in one of the scripts:

AVAILABLE_REGIONS=(
    australiaeast
    brazilsouth
    canadacentral
    centralindia
    centralus
    eastus
    eastus2
    francecentral
    japaneast
    northeurope
    norwayeast
    southcentralus
    southeastasia
    uksouth
    westeurope
    westus2
    westus3
)